src/Primitives.hs

```diff
@@ -632,6 +645,19 @@ primitiveMeta (XObj _ i _) ctx [XObj (Sym path _) _ _, XObj (Str key) _ _] = do
       return (evalError ctx
                         ("`meta` failed, I can’t find `" ++ show path ++ "`")
                         i)
+primitiveMeta (XObj _ i _) ctx [XObj (Sym path _) _ _, XObj (Str key) _ _] = do
```

Comment:
This looks almost identical to the case above, can we do some code sharing here?

Reply:
Good call! Done!